bridge/client/app/project-board/project-board.component.html

```diff
@@ -66,20 +86,50 @@ <h2 class="m-0 mt-1 mb-1" [textContent]="stage.stageName"></h2>
+                  <h2 class="m-0" [textContent]="service.serviceName"></h2>
+                  <ng-container *ngIf="openApprovals$ | async as openApprovals">
+                    <dt-icon class="icon2 out-of-sync-service" *ngIf="countOpenApprovals(openApprovals, selectedStage, service) > 0" name="deploy"></dt-icon>
+                  </ng-container>
+                  <ng-container *ngIf="project.getLatestDeployment(service, selectedStage) as deployment">
+                    <a *ngIf="deployment.data.deploymentURIPublic" [href]="deployment.data.deploymentURIPublic" target="_blank"><button dt-icon-button variant="nested"><dt-icon name="externallink"></dt-icon></button></a>
+                  </ng-container>
                 </div>
               </dt-info-group-title>
-              <p class="m-0 mt-1"><span [textContent]="deployment.data.image"></span>:<span [textContent]="deployment.data.tag"></span></p>
-              <p class="m-0 mb-1"><span class="bold">Deployed at: </span><span [textContent]="deployment.time | amCalendar:getCalendarFormats()"></span></p>
+              <ng-container *ngIf="project.getLatestDeployment(service, selectedStage) as deployment; else noDeployment">
+                <p class="m-0 mt-1"><span [textContent]="deployment.data.image"></span>:<span [textContent]="deployment.data.tag"></span></p>
+                <p class="m-0 mb-1"><span class="bold">Deployed at: </span><span [textContent]="deployment.time | amCalendar:getCalendarFormats()"></span></p>
+              </ng-container>
+              <ng-template #noDeployment>Not deployed yet.</ng-template>
             </dt-info-group>
           </ktb-expandable-tile-header>
+          <ng-container *ngIf="openApprovals$ | async as openApprovals">
+            <div *ngIf="countOpenApprovals(openApprovals, selectedStage, service) > 0; else noOutOfSyncDeployments">
+              <p class="m-0">Deployable artifacts for <span [textContent]="service.serviceName"></span> service</p>
+              <div fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="5px" *ngFor="let approval of getOpenApprovals(openApprovals, selectedStage, service)">
+                <h4 class="m-0 mt-1"><span [textContent]="approval.getShortImageName()"></span> (<span [textContent]="project.getDeploymentEvaluation(approval).data.result"></span>)</h4>
+                <button dt-icon-button (click)="approveDeployment(approval)" variant="nested" aria-label="Approve">
+                  <dt-icon class="success" name="checkmark"></dt-icon>
+                </button>
+                <button dt-icon-button (click)="declineDeployment(approval)" variant="nested" aria-label="Decline">
+                  <dt-icon class="error" name="abort"></dt-icon>
+                </button>
+              </div>
+            </div>
+          </ng-container>
+          <ng-template #noOutOfSyncDeployments>No pending deployments for this stage available.</ng-template>
         </ktb-expandable-tile>
```

Comment:
In my point of view this component has a lot content in it that could be extracted into smaller components. This selected part could extracted for example in a "deployment.component.ts".

Reply:
I created a follow-up issue for this, see #1989